lua/telescope/_extensions/coc.lua

```diff
@@ -382,7 +382,7 @@ local diagnostics = function(opts)
     end
   end
 
-  opts.hide_filename = utils.get_default(opts.hide_filename, true)
+  opts.path_display = utils.get_default(opts.path_display, true)
```

Comment:
The value of path_display is not boolean, https://github.com/nvim-telescope/telescope.nvim/wiki/Path-Display-Configuration

Reply:
Should be something like this: https://github.com/nvim-telescope/telescope.nvim/pull/839/files#diff-db4f373ad51427934e741b06da5a19d49a1c405ef7eaa5456b09cbf94a67c3b6